src/microbe_stage/MicrobeCamera.cs

```diff
@@ -117,24 +122,12 @@ public void ResolveNodeReferences()
             BackgroundPlane = GetNode<Spatial>("BackgroundPlane");
     }
 
-    public override void _UnhandledInput(InputEvent @event)
+    [RunOnAxis(new[] { "g_zoom_in", "g_zoom_out" }, new[] { -1.0f, 1.0f })]
+    public void Zoom(float delta, float value)
     {
-        bool changed = false;
-
-        if (@event.IsActionPressed("g_zoom_in", true))
-        {
-            CameraHeight -= ZoomSpeed;
-            changed = true;
-        }
-
-        if (@event.IsActionPressed("g_zoom_out", true))
-        {
-            CameraHeight += ZoomSpeed;
-            changed = true;
-        }
-
-        if (changed)
-            CameraHeight = CameraHeight.Clamp(MinCameraHeight, MaxCameraHeight);
+        _ = delta;
+        CameraHeight += ZoomSpeed * value;
```

Comment:
Should delta be multiplied into "value" here because otherwise the camera zoom speed is framerate dependent?

Reply:
I've kept the old system, which also was framerate dependent

Reply:
It's good time to fix this now.

Reply:
I think, assuming about 60 FPS, you an equivalent formula would be `value * delta * 55` 55 is from calculating how much you need to multiple 18 milliseconds by to get back to 1 second.